lib/pure/times.nim

```diff
@@ -219,17 +219,20 @@ proc getDayOfYear*(monthday: MonthdayRange, month: Month, year: int): YeardayRan
   else:
     result = daysUntilMonth[month] + monthday - 1
 
-proc getDayOfWeek*(monthday: MonthdayRange, month: Month, year: int): WeekDay {.tags: [], raises: [], benign .} =
-  ## Returns the day of the week enum from day, month and year.
-  ## Equivalent with ``initDateTime(day, month, year).weekday``.
-  assertValidDate monthday, month, year
-  # 1970-01-01 is a Thursday, we adjust to the previous Monday
-  let days = toEpochday(monthday, month, year) - 3
+proc getDayOfWeek*(epochday: int64): WeekDay {.tags: [], raises: [], benign .} =
+  ## Returns the day of the week enum from epoch day
```

Comment:
What's an "epoch day"?

Reply:
An epoch day is a date stored as the number of days since 1970-01-01